app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -352,12 +352,12 @@ function useGrantDetail() {
   const txHash = ref<string>();
 
   watch(
-    () => [grant.value, grantId.value, rounds.value, roundsCLRData.value, provider.value],
+    () => [grant.value, grantId.value, rounds.value, roundsCLRData.value, DEFAULT_PROVIDER],
```

Comment:
`DEFAULT_PROVIDER` should always hold a value, so we shouldn't need to check it here 👍

Reply:
Yep, and it's constant/not reactive/will never change, so we can remove it from the watcher entirely